src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+const totalSpan = document.querySelector('.total-population');
+const averageSpan = document.querySelector('.average-population');
+
+let calculatedTotal = 0;
+
+for (const population of populations) {
+  const number = Number(population.innerText.replaceAll(',', ''));
+
+  calculatedTotal += number;
```

Comment:
you can simplify it a bit
```suggestion
  calculatedTotal += +(population.innerText.replaceAll(',', ''));
```